### PR TITLE
README Revamp

### PR DESCRIPTION
**Pull request by refurbishing**
* fully revamped README using almost the same format as i did with the Suncord README
* fix releases URL

* fully on Markdown not like last time

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
+- Much more lightweight and faster than the official Discord app
+- Linux Screenshare with sound & wayland
+- Custom Splash animations from [this PR](https://github.com/Vencord/Vesktop/pull/355)
 
-## Windows contribution
+**Not yet supported**:
+- Global Keybinds
+  
 
-You will get test errors about venmic, you can ignore these as it's a linux only module.
```
Comment (verticalsync) [inline README.md:11]: Add this back into the installation part

File: README.md [hunk 2]
```diff
@@ -1,17 +1,30 @@
...
 
-## Installing
+Sunroof is a fork of [Vesktop](https://github.com/Vencord/Vesktop)
 
-Find the required installer you need [here](https://github.com/verticalsync/Sunroof/releases/tag/latest).
+**Main features**:
+- Suncord preinstalled
+- Much more lightweight and faster than the official Discord app
+- Linux Screenshare with sound & wayland
+- Custom Splash animations from [this PR](https://github.com/Vencord/Vesktop/pull/355)
```
Comment (verticalsync) [inline README.md:17]: Move this into a different header something like the suncord repo has about changes alongside vencord

**Review by refurbishing — COMMENTED** [review]
* logo and badges & good looking discord badge
* adds main features from Vesktop README but tweaked
* and using repo's sunroof logo PNG not catbox.moe like last time

File: README.md [hunk 3]
```diff
@@ -1,17 +1,30 @@
...
 
-Sunroof is a [Vesktop](https://github.com/Vencord/Vesktop) fork
+[![Sunroof](https://img.shields.io/badge/Suncord-orange?style=flat)](https://github.com/verticalsync/Suncord)
+[![Tests](https://github.com/verticalsync/Sunroof/actions/workflows/test.yml/badge.svg?branch=main)](https://github.com/verticalsync/Sunroof/actions/workflows/test.yml)
+[![Discord](https://img.shields.io/discord/1207691698386501634.svg?color=768AD4&label=Discord&logo=discord&logoColor=white)](https://discord.gg/VasF3Ma4Ab)
 
-## Installing
+Sunroof is a fork of [Vesktop](https://github.com/Vencord/Vesktop)
 
-Find the required installer you need [here](https://github.com/verticalsync/Sunroof/releases/tag/latest).
```
Comment (verticalsync) [inline README.md:7]: This never broke, and is the preferred way as latest in github goes to the most recently submitted

Comment (verticalsync) [reply README.md:7]: nvm, i misread it. It has to be `releases/latest` not `releases/tag/latest`

**Review by verticalsync — CHANGES_REQUESTED** [review]
.

**Review by verticalsync — APPROVED** [review]

**Pull request merged**